test/gtest/uct/test_p2p_err.cc

```diff
@@ -72,8 +72,7 @@ class uct_p2p_err_test : public uct_p2p_test {
             case OP_AM_SHORT:
                 status = uct_ep_am_short(sender_ep(), am_id, 0, buffer, length);
                 break;
-            case OP_AM_SHORT_IOV:
-            {
+            case OP_AM_SHORT_IOV: {
```

Comment:
no need for `{}`

Reply:
Braces are required because there is a variable declaration inside a macro.